### ISSUE TITLE
Custom AppXManifest.xml template

### ISSUE THREAD

**Issue by evanwalsh** [electron-userland/electron-builder#3072]
* **Version**: 20.15.1

* **Target**: Windows x64 AppX

Hello! I'm a big fan of the project, but I've found myself in need of supplying a custom AppXManifest.xml. Mostly, I need to add an extension to the manifest to [support `electron-winstore-auto-launch`](https://github.com/felixrieseberg/electron-winstore-auto-launch). I don't see a way to currently do this.

I'm open to adding the feature myself, but I wanted to drop a line here before I tried to make a pull request.

**Comment by mccxiv** [comment]
Hey @evanwalsh , did you figure out any way to achieve this? I also need to add the auto start extension

**Comment by develar** [comment]
You can expect this feature today.

**Comment by develar** [comment]
20.21.0

**Comment by aschultz** [comment]
It'd be great if we could get the larger ask (custom AppxManifest.xml template) implemented. That would allow devs to add Extension elements, specify the target Windows versions, and add any new elements Microsoft may introduce without requiring electron-builder explicitly add them.

**Comment by mvbn** [comment]
I agree, for example, I'm interested to have the app name on the tiles icons.

But when Microsoft will allow to enable the dark mode (for the window title bar, etc) to all apps (not only WPF), I will be interested also on this.

**Comment by jameshfisher** [comment]
Wondering whether there's any progress here - I'd also like to be able to pass a custom `AppxManifest.xml`, in order to work around this bug where electron-builder generates an invalid `Publisher` string: https://github.com/electron-userland/electron-builder/issues/3501

**Comment by mmaietta** [comment]
https://github.com/electron-userland/electron-builder/pull/8609
